samples/c_cxx/README.md

```diff
@@ -16,23 +18,47 @@ You may get a precompiled libpng library from [https://onnxruntimetestdata.blob.
 ## Install ONNX Runtime
 You may either get a prebuit onnxruntime from nuget.org, or build it from source by following the [build instructions](../../BUILD.md). 
 If you build it by yourself, you must append the "--build_shared_lib" flag to your build command. 
+Open Developer Command Prompt for Visual Studio version you are going to use. This will setup necessary environment for the compiler and other things to be found.
+```
+build.bat --config RelWithDebInfo --build_shared_lib --parallel 
+```
+
+By default this will build a project with "C:\Program Files (x86)\onnxruntime" install destination. This is a protected folder on Windows. If you do not want to run installation with elevated priviliges you will need to override the default installation location by passing extra CMake arguments. For example:
+
 ```
-build.bat --config RelWithDebInfo --build_shared_lib --parallel
+build.bat --config RelWithDebInfo --build_shared_lib --parallel  --cmake_extra_defines CMAKE_INSTALL_PREFIX=c:\dev\ort_install
 ```
-When the build is done, run Visual Studio as administrator and open the onnxruntime.sln file in your build directory.
-![vs.png](vs.png)
 
-When the solution is loaded, change the build configuration to "RelWithDebInfo"(which must match your previous build command), then select the "INSTALL" project, and build it.  It will install your onnxruntime to  "C:\Program Files (x86)\onnxruntime"
+By default products of the build on Windows go to .\build\Windows\<config> folder. In the case above it would be .\build\Windows\RelWithDebInfo.
+If you did not specify alternative installation location above you would need to open an elevated command prompt to install onnxruntime.
+Run the following commands.
+
+```
+cd .\Windows\RelWithDebInfo
+msbuild INSTALL.vcxproj /p:Configuration=RelWithDebInfo
+```
 
 ## Build the samples
-Open cmd.exe, change your current directory to samples\c_cxx, then run
+
+Open Open Developer Command Prompt for Visual Studio version you are going to use, change your current directory to samples\c_cxx, then run
 ```bat
-mkdir build
-cmake .. -A x64 -T host=x64 -DLIBPNG_ROOTDIR=C:\path\to\your\libpng\binary
+mkdir build && cd build
+cmake .. -A x64 -T host=x64 -DLIBPNG_ROOTDIR=C:\path\to\your\libpng\binary -DONNXRUNTIME_ROOTDIR=c:\dev\ort_install
 ```
-You may omit the "-DLIBPNG_ROOTDIR=..." argument if you don't have the libpng library.     
+You may omit the "-DLIBPNG_ROOTDIR=..." argument if you don't have the libpng library.
+You may omit "-DONNXRUNTIME_ROOTDIR=..." if you installed to a default location.
+
```

Comment:
Is this because the default install adds something to PATH? If so it's not that you installed to a default location, it's that you installed using INSTALL.vcxproj.